tests/config-array.test.js

```diff
@@ -1105,46 +1105,71 @@ describe('ConfigArray', () => {
 					.throw(/normalized/);
 			});
 
+		});
+
+		describe('isFileConfigured()', () => {
+
+			it('should throw an error when not normalized', () => {
+				const filename = path.resolve(basePath, 'foo.js');
+
+				expect(() => {
+					unnormalizedConfigs.isFileConfigured(filename);
+				})
+					.to
+					.throw(/normalized/);
+			});
+
+		});
+
+		describe('isFileIgnored() / isFileConfigured()', () => {
```

Comment:
I've added assertions for `isFileConfigured` to those of `isFileIgnored`, because in ESLint `isFileConfigured` will be called after `isFileIgnored` returns `true`, so these methods go in pair. Alternatively, we could duplicate some of test cases to keep the assertions separated.